shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/LeafFormat.kt

```diff
@@ -10,6 +10,14 @@ sealed class LeafFormat {
 
     abstract fun noPredictionsStatus(): UpcomingFormat.NoTripsFormat?
 
+    val hidePredictions: Boolean
```

Comment:
suggestion: It may help w/ clarity for this variable name to reflect what it represents directly, something like `isAllServiceDisrupted`. `hidePredictions` doesn't capture that this would also hide schedules for say a branched CR route. It also seems more named based on the UI-related effects, which I don't think the data representation should be aware of.